cfme/tests/v2v/test_migration_throttling.py

```diff
@@ -0,0 +1,91 @@
+import fauxfactory
+import pytest
+
+from widgetastic.widget import NoSuchElementException
+
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.fixtures.provider import rhel69_template
+from cfme.fixtures.provider import rhel7_minimal
+from cfme.fixtures.provider import ubuntu16_template
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.markers.env_markers.provider import ONE_PER_VERSION
+
+pytestmark = [
+    test_requirements.v2v,
+    pytest.mark.provider(
+        classes=[OpenStackProvider, RHEVMProvider],
+        selector=ONE_PER_VERSION,
+        required_flags=["v2v"],
+        scope="module"
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_TYPE,
+        fixture_name="source_provider",
+        required_flags=["v2v"],
+        scope="module",
+    ),
+    pytest.mark.usefixtures("v2v_provider_setup"),
+]
+
+
+@pytest.mark.parametrize(
+    "mapping_data_multiple_vm_obj_single_datastore",
```

Comment:
Can this variable name be shortened? Why not just call it `data`?

Reply:
@sshveta Any thoughts on this? My main concern is code readability.

Reply:
@john-dupuy : There are lot of fixtures like these in v2v_fixtures.py with different data set like mapping_data_single_datastore, mapping_data_single_network, mapping_data_dual_vm_obj_dual_datastore, 
mapping_data_vm_obj_dual_nics and they create data as their name suggest .

Reply:
Got it, that's okay :+1: